src/components/content/heroes/blog-hero.tsx

```diff
@@ -44,28 +43,16 @@ const Attribution = ({ attribution }) => (
  */
 export const BlogHero = ({
   image,
-  imageSquare,
   attribution,
   title,
   children,
   kicker,
 }: ImageHeroProps) => {
```

Comment:
Please check, if any types need updating. I see that you deleted most occurences of actual code, but couldn't find any types that were delete.